ecs-cli/modules/cli/compose/project/project_parseV1V2_test.go

```diff
@@ -0,0 +1,377 @@
+		},
+	}
+	workingDir := "/var"
+
+	composeV1FileString := `web:
+  cpu_shares: 73
+  command:
+   - bundle exec thin -p 3000
+  dns:
+   - 1.2.3.4
+  dns_search: search.example.com
+  entrypoint: /code/entrypoint.sh
+  environment:
+    RACK_ENV: development
+  extra_hosts:
+   - test.local:127.10.10.10
+  hostname: "foobarbaz"
+  image: web
+  labels:
+   - label1
+   - com.foo.label2=value
+  links:
+   - "redis:redis"
+  log_driver: json-file
+  log_opt:
+    max-file: 50
+    max-size: 50k
+  mem_limit: 1000000000
+  ports:
+   - '5000:5000'
+   - "127.0.0.1:8001:8001"
+  privileged: true
+  read_only: true
+  security_opt:
+   - label:type:test_virt
+  shm_size: 128M
+  ulimits:
+    nofile: 1024
+  user: user
+  volumes:
+   - ./code
+  working_dir: /var
+redis:
+  image: redis
+  volumes_from:
+    - web:ro`
+
+	// Setup docker-compose file
+	tmpfile, err := ioutil.TempFile("", "test")
+	assert.NoError(t, err, "Unexpected error in creating test file")
+
+	defer os.Remove(tmpfile.Name())
+
+	_, err = tmpfile.Write([]byte(composeV1FileString))
+	assert.NoError(t, err, "Unexpected error in writing to test file")
+
+	err = tmpfile.Close()
+	assert.NoError(t, err, "Unexpected error closing file")
+
+	// Set up project
+	project := setupTestProject(t)
+	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile.Name())
+
+	// Parse docker compose file
+	actualConfigs, err := project.parseV1V2()
+	assert.NoError(t, err, "Unexpected error parsing file")
+
+	// verify redis ContainerConfig
+	redis, err := getContainerConfigByName("redis", actualConfigs)
+	assert.NoError(t, err, "Unexpected error retrieving redis config")
+
+	assert.Equal(t, redisImage, redis.Image, "Expected image to match")
+	assert.Equal(t, volumesFrom, redis.VolumesFrom, "Expected VolumesFrom to match")
+
+	// verify web ContainerConfig
+	web, err := getContainerConfigByName("web", actualConfigs)
+	assert.NoError(t, err, "Unexpected error retrieving web config")
+
+	assert.Equal(t, command, web.Command, "Expected Command to match")
+	assert.Equal(t, cpuShares, web.CPU, "Expected CPU to match")
+	assert.Equal(t, dnsSearchDomains, web.DNSSearchDomains, "Expected DNSSearchDomains to match")
+	assert.Equal(t, dnsServers, web.DNSServers, "Expected DNSServers to match")
+	assert.Equal(t, labels, web.DockerLabels, "Expected docker labels to match")
+	assert.Equal(t, securityOpts, web.DockerSecurityOptions, "Expected DockerSecurityOptions to match")
+	assert.Equal(t, entryPoint, web.Entrypoint, "Expected EntryPoint to be match")
+	assert.Equal(t, env, web.Environment, "Expected Environment to match")
+	assert.Equal(t, extraHosts, web.ExtraHosts, "Expected ExtraHosts to match")
+	assert.Equal(t, hostname, web.Hostname, "Expected Hostname to match")
+	assert.Equal(t, webImage, web.Image, "Expected Image to match")
+	assert.Equal(t, links, web.Links, "Expected Links to match")
+	assert.Equal(t, logging, web.LogConfiguration, "Expected LogConfiguration to match")
```

Comment:
oh weird, the assert still works - is libcompose returning a `logging` struct still?

Reply:
No, it's because `parseV1V2()`, which is the function I care about testing, converts things into a log configuration (see previous comment)